var/spack/repos/builtin/packages/aml/package.py

```diff
@@ -3,66 +3,154 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
-import os
-
 from spack import *
 
 
 class Aml(AutotoolsPackage):
-    """AML: Building Blocks for Memory Management."""
+    """
+    AML: Building Blocks for Memory Management.
+
+    For information on AML please visit AML homepage.
+
+    Possible variants for aml spack package are:
+    - `+cuda`: enables library features implementation over cuda runtime library.
+    All cuda variants from the spack package "cuda" are allowed.
+    - `+hip`: enables library features implementation over hip runtime library.
+    All variants from the spack package "hip" are allowed.
+    - `+ze`: enables library features implementation over intel level_zero
+    library. This feature relies on aml 'automagick' detection until a spack
+    package becomes available.
+    - `+hwloc`: enables library features implementation over hwloc library.
+    If `+hwloc` variant is used, `hwloc@2.0:` is used as a dependency for
+    building hwloc features.
+    - `+opencl`: enables library features implementation over opencl library.
+    If `+rocm` and `+opencl` are used together, `rocm-opencl` is used as a
+    dependency for building opencl features.
+    - `+all`: enables all above variants.
+    - `openmp-targets={spir64,}`: Enable interoperability of pointers between
+    intel OpenMP and intel level_zero for spir64 openmp targets.
+    - `hip-platform={amd,nvidia}`: Compile aml with this hip backend
+    implementation. Under the hood, hip calls either amd or cuda functions.
+
+    Examples:
+    ========
+
+    - spack install aml +cuda +hwloc +hip +ze +opencl
+    - spack install aml +cuda +hip hip-platform=nvidia
+    - spack install aml +ze openmp-targets=spir64
+    """
 
     homepage = "https://argo-aml.readthedocs.io/"
-    url = "https://www.mcs.anl.gov/research/projects/argo/downloads/aml-0.1.0.tar.gz"
-    git = "https://github.com/anlsys/aml.git"
+
     maintainers = ['perarnau']
 
     test_requires_compiler = True
 
     tags = ['e4s']
 
-    version('0.1.0', sha256='cc89a8768693f1f11539378b21cdca9f0ce3fc5cb564f9b3e4154a051dcea69b')
-    version('master', branch='master', submodules=True)
-
-    depends_on('numactl')
-
+    # Package sources
+    ###################################
+
+    url = 'https://github.com/anlsys/aml/releases/download/v0.2.0/aml-0.2.0.tar.gz'
+    git = 'https://github.com/anlsys/aml.git'
+
+    # version string is generated from git tags, requires entire repo
+    version('master', branch='master', submodules=True, get_full_repo=True)
+
+    version('0.2.0', sha256='eea44780c11cff3f585f7ad1bc4fcc0d9d46aa8c27de234f6ae3703c9bb57684')
```

Comment:
```suggestion
    version('0.2.0', sha256='2044a2f3f1d7a19827dd9c0726172b690189b4d3fe938656c4160c022468cc4a')
```
You'll find this in the log of the failed GitLab CI job.